app/controllers/single_use_links_viewer_controller.rb

```diff
@@ -17,53 +14,7 @@ def initialize(user, single_use_link)
       can :read, ActiveFedora::Base do |obj|
         single_use_link.valid? && single_use_link.itemId == obj.id && single_use_link.destroy!
       end if single_use_link
-    end
```

Comment:
Hmmm, we might want to put this in the behavior like @cam156 suggested, but can we also put it behind a method that downstream adopters can override? I suspect that's why @weiweishi left this out of the controller, so that this isn't foisted on all adopters.